CHANGELOG.D/1560.feature

```diff
@@ -0,0 +1,3 @@
+Added the `--exclude-from-files` option in `neuro cp`.
+
+API: Added the optional parameter `ignore_file_names` in the `upload_dir()` methods of `storage` and `blob_storage`.
```

Comment:
This text is correct but please note: `towncrier` doesn't respect new lines but merges the whole file's text into the single list bullet.